### PR TITLE
Add inital 1.9.0 release note

### PR DESCRIPTION
**Pull request by yongtang**
Add inital 1.9.0 release note, will update later if there are
more items to add.

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

File: notes/coredns-1.9.0.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Brought to You By
+
+Chris O'Haver,
+Ondřej Benkovský,
+Yong Tang,
+xuweiwei
+
+## Noteworthy Changes
+
```
Comment (chrisohaver) [inline notes/coredns-1.9.0.md:24]:
```suggestion

* plugin/kubernetes: remove wildcard query functionality (https://github.com/coredns/coredns/pull/5019)
* Health-checks should respect force_tcp (https://github.com/coredns/coredns/pull/5109)
```suggestion

File: notes/coredns-1.9.0.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+description = "CoreDNS-1.9.0 Release Notes."
+tags = ["Release", "1.9.0", "Notes"]
+release = "1.9.0"
+date = "2022-02-01T00:00:00+00:00"
+author = "coredns"
++++
+
+This is a release with bug fixes and some new features added. Starting with 1.9.0
+the minimal required go version will be 1.17.
+
```
Comment (chrisohaver) [inline notes/coredns-1.9.0.md:14]:
```suggestion
Wildcard queries are no longer supported by the _kubernetes_ plugin.


```

File: notes/coredns-1.9.0.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+## Brought to You By
+
+Chris O'Haver,
+Ondřej Benkovský,
```
Comment (chrisohaver) [inline notes/coredns-1.9.0.md:18]:
```suggestion
Ondřej Benkovský,
Tomas Hulata,
```

File: notes/coredns-1.9.0.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+Tomas Hulata,
+Yong Tang,
+xuweiwei
+
+## Noteworthy Changes
+
+* plugin/kubernetes: remove wildcard query functionality (https://github.com/coredns/coredns/pull/5019)
+* Health-checks should respect force_tcp (https://github.com/coredns/coredns/pull/5109)
+```suggestion
```
Comment (stevo-f3) [inline notes/coredns-1.9.0.md:27]: @yongtang this line with suggestion is breaking up release notes rendering

Comment (chrisohaver) [reply notes/coredns-1.9.0.md:27]: My bad.  my browser kept adding extraneous `suggestion` markdowns when using the "suggestion" feature. I thought I cleaned them all up, but missed one 

* #5175

**Pull request merged**